addons/account/tests/test_account_move_reconcile.py

```diff
@@ -3532,13 +3532,13 @@ def test_reconcile_cash_basis_workflow_multi_currency(self):
         caba_transition_exchange_moves_1 = caba_transition_lines_1.matched_credit_ids.exchange_move_id
         self.assertEqual(len(caba_transition_exchange_moves_1), 2)
         self.assertRecordValues(caba_transition_exchange_moves_1[0].line_ids, [
```

Comment:
Before this commit, during the test there is two exchange moves for 2 partials, the index to where the exchange move need to be linked is the 0 and 1. From my research the second exchange move shouldn't be linked to the second partial since the reconciled lines ids have nothing in common. i think this make more sense